wdl2cwl/tests/oop_cwl_files/bcftools_stats.cwl

```diff
@@ -0,0 +1,170 @@
+    type:
+      - string
+      - 'null'
+  - id: regions
+    type:
+      - string
+      - 'null'
+  - id: regionsFile
+    type:
+      - File
+      - 'null'
+  - id: samplesFile
+    type:
+      - File
+      - 'null'
+  - id: targets
+    type:
+      - string
+      - 'null'
+  - id: targetsFile
+    type:
+      - File
+      - 'null'
+  - id: userTsTv
+    type:
+      - string
+      - 'null'
+  - id: threads
+    default: 0
+    type: int
+  - id: memory
+    default: 256M
+    type: string
+  - id: dockerImage
+    default: quay.io/biocontainers/bcftools:1.10.2--h4f4756c_2
+    type: string
+outputs:
+  - id: stats
+    type: File
+    outputBinding:
+        glob: "$(inputs.outputPath === null ? (inputs.inputVcf.basename + '.stats')\
+            \ : inputs.outputPath)"
+requirements:
+  - class: DockerRequirement
+    dockerPull: quay.io/biocontainers/bcftools:1.10.2--h4f4756c_2
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: example.sh
+        entry: |4
+
+            set -e
+            mkdir -p \$(dirname  $(inputs.outputPath === null ? inputs.inputVcf.basename + '.stats' : inputs.outputPath))
+            bcftools stats \
+            $(inputs.afBins === null ? "" : "--af-bins " inputs.afBins) \
+            $(inputs.afTag === null ? "" : "--af-tag " inputs.afTag) \
+            $(inputs.firstAlleleOnly ? "--1st-allele-only" : "") \
+            $(inputs.collapse === null ? "" : "--collapse " inputs.collapse) \
+            $(inputs.depth === null ? "" : "--depth " inputs.depth) \
+            $(inputs.exclude === null ? "" : "--exclude " inputs.exclude) \
+            $(inputs.exons === null ? "" : "--exons " inputs.exons) \
+            $(inputs.applyFilters === null ? "" : "--apply-filters " inputs.applyFilters) \
+            $(inputs.fastaRef === null ? "" : "--fasta-ref " inputs.fastaRef) \
+            $(inputs.include === null ? "" : "--include " inputs.include) \
+            $(inputs.splitByID ? "--split-by-ID" : "") \
+            $(inputs.regions === null ? "" : "--regions " inputs.regions) \
+            $(inputs.regionsFile === null ? "" : "--regions-file " inputs.regionsFile) \
+            $(inputs.samples.map(function(el) {return el.path}).join(",")) \
+            $(inputs.samplesFile === null ? "" : "--samples-file " inputs.samplesFile) \
+            $(inputs.targets === null ? "" : "--targets " inputs.targets) \
+            $(inputs.targetsFile === null ? "" : "--targets-file " inputs.targetsFile) \
+            $(inputs.userTsTv === null ? "" : "--user-tstv " inputs.userTsTv) \
+            --threads $(inputs.threads) \
+            $(inputs.verbose ? "--verbose" : "") \
+            $(inputs.inputVcf)$(inputs.compareVcf) > $(inputs.outputPath === null ? inputs.inputVcf.basename + '.stats' : inputs.outputPath)
+  - class: InlineJavascriptRequirement
+  - class: NetworkAccess
+    networkAccess: true
+  - class: ResourceRequirement
+    ramMin: inputs.threads + 1
```

Comment:
double with then next line? Also, not a float nor a CWL expression

Reply:
Sorry about that.